pkg/network/setup/hotplugcontroller.go

```diff
@@ -0,0 +1,41 @@
+package network
+
+import (
+	v1 "kubevirt.io/api/core/v1"
+)
+
+func NetworksToHotplug(vmi *v1.VirtualMachineInstance) []v1.Network {
+	var networksToHotplug []v1.Network
+	indexedIfacesFromStatus := indexedInterfacesFromStatus(vmi.Status.Interfaces, selectAll())
+	indexedNetsFromSpec := indexedNetworksFromSpec(vmi.Spec.Networks)
+	for ifaceName, iface := range indexedNetsFromSpec {
+		if _, wasFound := indexedIfacesFromStatus[ifaceName]; !wasFound {
+			networksToHotplug = append(networksToHotplug, iface)
+		}
+	}
+	return networksToHotplug
+}
+
+func selectAll() func(v1.VirtualMachineInstanceNetworkInterface) bool {
+	return func(v1.VirtualMachineInstanceNetworkInterface) bool {
+		return true
+	}
+}
+
+func indexedInterfacesFromStatus(interfaces []v1.VirtualMachineInstanceNetworkInterface, p func(ifaceStatus v1.VirtualMachineInstanceNetworkInterface) bool) map[string]v1.VirtualMachineInstanceNetworkInterface {
```

Comment:
I'm not clear why you need the predicate at the moment. The `selectAll` is not useful for now, perhaps add it when it will be needed.